### ISSUE TITLE
SC Content on Hover or Focus: Hover/Focus to Hoverable/Persistent

### ISSUE THREAD

**Issue by steverep** [w3c/wcag21#565]
As I was writing the [draft Understanding for SC Content on Hover or Focus](https://rawgit.com/w3c/wcag21/content-on-hover-or-focus/understanding/21/content-on-hover-or-focus.html), it became very clear that the original intent for the "Focus" condition as described in #75 has gotten lost with language changes prior to the original CFC.  The same is true to a lesser extent for the "Hover" condition.  This is explained below with a proposal to fix the SC.

#### Focus to Persistent
The original intent of this SC was described as follows in #75:

> Finally, when a popup contains user interface components that can gain focus, a user technique can be employed to move focus onto the popup. As long as the popup remains displayed, the view can then be magnified, scrolled, or panned for optimal perception without regard to mouse position over the content. For popups with multiple clickable items, this also offers the advantage of being able to make and correct a clicking mistake without losing visibility of the popup.

This intent has been completely lost by the mysterious removal of any language referring to the components of the additional content itself having focus (i.e. it now only mentions the trigger).  However, I now believe it is unnecessary to state this since it is covered by SC 2.1.1 Keyboard.

The remaining intent though is the persistence of the additional content once it has been triggered.  The original idea was that this would be implied by both the Hover and Focus bullets., but I think it is better to serve this under a single distinct bullet as follows:

**Persistent:** The additional content remains visible until the user dismisses it or its information is no longer valid.

Basically, we just remove any reference to how the content was triggered since it applies to both hover and focus.  The added language about the information no longer being valid is meant to resolve #491.

#### Hover to Hoverable
The original intent of the Hover condition was to imply that the content would be persistent as well as hoverable.  I think that point is definitely lost with the language in the draft.  I propose just leaving it as is and renaming the bullet handle to "Hoverable", and allowing the new Persistent condition to cover how it can disappear.
